test/stateless_escrow_timed_lock_test.py

```diff
@@ -28,7 +28,7 @@ def client(test_config):
     return client
 
 
-@pytest.fixture(scope='class')
+#@pytest.fixture(scope='class')

```

Comment:
this wallet needs to persists across all the tests in this file

Reply:
disregard my last comment....seems like we aren't using wallet_2 at all ( I believe when I was originally figuring out this code I had intended for tests that involve wallet_2)
Let's just remove the wallet_2 function all together